examples/benchmarks/input_output/dyck_one.effekt

```diff
@@ -37,17 +37,19 @@ def size(tree: Tree): Int =
     }
   }
 
+extern def filename(): String =
+  js { "/tmp/dyck_one_js.txt" }
+  llvm { "/tmp/dyck_one_llvm.txt" }
+
```

Comment:
Check if this is necessary and undo it if not. Its purpose was to prevent multiple test processes (for the different backends) accessing the same file.

Reply:
Could we also add some `EFFEKT_BACKEND` constant such that we can write `def filename(): String = "/tmp/dyck_one_${EFFEKT_BACKEND}.txt"` using the new string interpolation? This would reduce code duplication and make it easier to add new backends in the future.

Reply:
We do not have `extern val`s but it is easy to have a function somewhere:

```
extern def EFFEKT_BACKEND(): String =
  js { "js" }
  llvm { "llvm" }
  chez { "chez" }
   ...
```

Reply:
I am not sure we want to use string interpolation in this particular scenario, since this pulls a lot of different dependencies and @phischu would be very unhappy.